core/src/main/scala/cats/collections/Predicate.scala

```diff
@@ -63,12 +63,20 @@ object Predicate extends PredicateInstances {
 }
 
 trait PredicateInstances {
+  implicit def predicateContravariantMonoidal: ContravariantMonoidal[Predicate] = new ContravariantMonoidal[Predicate] {
+    override def contramap[A, B](fb: Predicate[A])(f: B => A): Predicate[B] =
+      Predicate(f andThen fb.apply)
+    override def product[A, B](fa: Predicate[A], fb: Predicate[B]): Predicate[(A, B)] =
+      Predicate(v => fa(v._1) || fb(v._2))
+    override def unit: Predicate[Unit] = Predicate.empty
+  }
+
   implicit def predicateMonoid[A]: Monoid[Predicate[A]] = new Monoid[Predicate[A]] {
     override def empty: Predicate[A] = Predicate.empty
     override def combine(l: Predicate[A], r: Predicate[A]): Predicate[A] = l union r
   }
 
-  implicit val predicateInstance: MonoidK[Predicate] = new MonoidK[Predicate] {
+  implicit val predicateMonoidK: MonoidK[Predicate] = new MonoidK[Predicate] {
```

Comment:
Dunno if this rename is cool from a bincompat perspective.

Reply:
There are no guarantees about that yet, so it's fine.

Reply:
I think we should add mima checking sooner than later.

I personally really avoid libraries that don't try to minimize breaks (and in fact, despite contributing to this library, I am afraid to depend on it).

Reply:
Happy to call it 1.0 at some point in the near future.